.github/workflows/test.yaml

```diff
@@ -14,10 +14,6 @@ jobs:
         go-version: [1.22.x]
         os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
do we need ubuntu-latest twice (here and down there in include) ? or we can leave it for the future

Reply:
I think this problem will be solved when we merge sqlite PR, I will remove my changes here